### ISSUE TITLE
Ping possible co-authors

### ISSUE THREAD

**Issue by jklymak** [jklymak/OtisTidalPrediction#1]
ping @mrayson and @ofringer

I pulled your tide stuff out of suntans for this, and modified to have fewer internal dependencies, but otherwise its the same.  Is it OK?  And/or would you like to be added as collaborators?

**Comment by ofringer** [comment]
Which part did you use? No need to add me as collaborator but Matt might be
interested.
Oliver

**Comment by jklymak** [comment]
Sorry.  Just the tide prediction for translating OTIS to actual predictions.

**Comment by mrayson** [comment]
Yes, please add me as a collaborator @jklymak. The tide prediction code is
better placed in its own repository.

**Comment by jklymak** [comment]
I can prob make you ge owner if you’d like.  The changes I made were pretty minimal.

**Comment by mrayson** [comment]
An acknowledgement and link to the initial code would be fine. Happy to
collaborate if you want to expand the package beyond a simple OTIS
interface tool though.
